2021-12-13 19:53:36,205 INFO    MainThread:6020 [wandb_setup.py:_flush():71] setting env: {}
2021-12-13 19:53:36,205 INFO    MainThread:6020 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-13 19:53:36,205 INFO    MainThread:6020 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_195336-2rx56uf8\logs\debug.log
2021-12-13 19:53:36,205 INFO    MainThread:6020 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_195336-2rx56uf8\logs\debug-internal.log
2021-12-13 19:53:36,206 INFO    MainThread:6020 [wandb_init.py:init():404] calling init triggers
2021-12-13 19:53:36,206 INFO    MainThread:6020 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-12-13 19:53:36,207 INFO    MainThread:6020 [wandb_init.py:init():449] starting backend
2021-12-13 19:53:36,207 INFO    MainThread:6020 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2021-12-13 19:53:36,211 INFO    MainThread:6020 [backend.py:ensure_launched():199] starting backend process...
2021-12-13 19:53:36,293 INFO    MainThread:6020 [backend.py:ensure_launched():204] started backend process with pid: 15240
2021-12-13 19:53:36,293 INFO    MainThread:6020 [wandb_init.py:init():458] backend started and connected
2021-12-13 19:53:36,295 INFO    MainThread:6020 [wandb_init.py:init():519] updated telemetry
2021-12-13 19:53:36,317 INFO    MainThread:6020 [wandb_init.py:init():549] communicating current version
2021-12-13 19:53:36,883 INFO    MainThread:6020 [wandb_init.py:init():554] got version response 
2021-12-13 19:53:36,883 INFO    MainThread:6020 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-13 19:53:37,142 INFO    MainThread:6020 [wandb_init.py:init():592] starting run threads in backend
2021-12-13 19:53:37,234 INFO    MainThread:6020 [wandb_run.py:_console_start():1816] atexit reg
2021-12-13 19:53:37,235 INFO    MainThread:6020 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2021-12-13 19:53:37,235 INFO    MainThread:6020 [wandb_run.py:_redirect():1727] Wrapping output streams.
2021-12-13 19:53:37,238 INFO    MainThread:6020 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-13 19:53:37,238 INFO    MainThread:6020 [wandb_init.py:init():619] run started, returning control to user process
2021-12-13 19:53:37,238 INFO    MainThread:6020 [wandb_config.py:__setitem__():144] config set learning_rate = 0.001 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000017464F355E0>>
2021-12-13 19:53:37,238 INFO    MainThread:6020 [wandb_run.py:_config_callback():962] config_cb learning_rate 0.001 None
2021-12-13 19:53:37,239 INFO    MainThread:6020 [wandb_config.py:__setitem__():144] config set batch_size = 32 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000017464F355E0>>
2021-12-13 19:53:37,239 INFO    MainThread:6020 [wandb_run.py:_config_callback():962] config_cb batch_size 32 None
2021-12-13 19:53:51,491 INFO    MainThread:6020 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 255
2021-12-13 19:53:51,491 INFO    MainThread:6020 [wandb_run.py:_restore():1758] restore
2021-12-13 19:53:52,039 INFO    MainThread:6020 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 19:53:52,158 INFO    MainThread:6020 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 19:53:52,411 INFO    MainThread:6020 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 19:53:53,302 INFO    MainThread:6020 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 3424
}

2021-12-13 19:53:53,419 INFO    MainThread:6020 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 3540
}

2021-12-13 19:53:53,530 INFO    MainThread:6020 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 3540
}

2021-12-13 19:53:53,638 INFO    MainThread:6020 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3540
  total_bytes: 3540
}

2021-12-13 19:53:53,747 INFO    MainThread:6020 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3540
  total_bytes: 3540
}

2021-12-13 19:53:53,853 INFO    MainThread:6020 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3540
  total_bytes: 3540
}

2021-12-13 19:53:53,963 INFO    MainThread:6020 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3540
  total_bytes: 3540
}

2021-12-13 19:53:54,075 INFO    MainThread:6020 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3540
  total_bytes: 3540
}

2021-12-13 19:53:54,312 INFO    MainThread:6020 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3540
  total_bytes: 3540
}

2021-12-13 19:53:54,841 INFO    MainThread:6020 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3540
  total_bytes: 3540
}
local_info {
}

2021-12-13 19:53:55,904 INFO    MainThread:6020 [wandb_run.py:_append_history():2134] rendering history
2021-12-13 19:53:55,904 INFO    MainThread:6020 [wandb_run.py:_append_summary():2089] rendering summary
2021-12-13 19:53:55,904 INFO    MainThread:6020 [wandb_run.py:_append_files():2184] logging synced files
